litmus-portal/frontend/src/views/Analytics/ApplicationDashboards/Steps/TuneTheQueries/EditPanelsWizard/index.tsx

```diff
@@ -496,10 +500,9 @@ const EditPanelsWizard: React.FC<EditPanelsWizardProps> = ({
           ))}
           <StyledTab
             label={t(
-              'analyticsDashboard.applicationDashboards.tuneTheQueries.addMetric'
+              'analyticsDashboard.applicationDashboards.tuneTheQueries.addPanel'
             )}
             {...a11yProps(dashboardDetails.selectedPanels?.length)}
-            key="tab-addMetric"
```

Comment:
This was done as a fix for a codacy issue can you verify the same ?

Reply:
No codacy issue related to this was found. Please do verify and check the same